site/docs/v1/tech/apis/themes.adoc

```diff
@@ -117,6 +118,74 @@ include::docs/v1/tech/apis/_theme-responses-body.adoc[]
 include::docs/v1/tech/apis/_theme-responses-body-suffix.adoc[]
 {%capture singleResponse %}false{% endcapture %}
 
+== Search for Themes
+[NOTE.since]
+====
+This API has been available since 1.44.0
+====
+
+This API is used to search for Themes and may be called using the `GET` or `POST` HTTP methods. Examples of each are provided below. The `POST` method is provided to allow for a richer request object without worrying about exceeding the maximum length of a URL. Calling this API with either the `GET` or `POST` HTTP method will provide the same search results given the same query parameters.
+
+=== Request
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Themes by name
+[.endpoint]
+.URI
+--
+[method]#GET# [uri]#/api/theme/search``?name=\{name\}``#
+--
+
+==== Request Parameters
+
+:parameter_prefix:
+include::docs/v1/tech/apis/_theme-search-request-parameters.adoc[]
+:parameter_prefix!:
+
+[.api-authentication]
+link:/docs/v1/tech/apis/authentication#api-key-authentication[icon:lock[type=fas]] Search for Themes using the given search and sort criteria
```

Comment:
Not entirely an accurate title, since there are no sort criteria provided. Maybe 

`Search for Themes using the given search criteria`

?

Reply:
I think this works here since it's for the POST request. The example includes sort criteria.